module/VuFindConsole/src/VuFindConsole/Command/ScheduledSearch/NotifyCommandFactory.php

```diff
@@ -88,5 +89,7 @@ public function __invoke(
             $container->get(\VuFind\I18n\Locale\LocaleSettings::class),
             ...($options ?? [])
         );
+        $command->setPathResolver($container->get(PathResolver::class));
```

Comment:
I see changes to the factory and the test but not to NotifyCommand itself; are these changes still relevant/needed? Am I missing something?

Reply:
Never mind, I see that this is related to LanguageInitializerTrait; makes sense to me now! :-)